packages/polymath-js/src/contracts/TickerRegistry.js

```diff
@@ -119,7 +119,7 @@ class TickerRegistry extends Contract {
         swarmHash
       ),
       null,
-      1.15
+      1.05
```

Comment:
Is it safe to lower this buffer?

Reply:
That was the original value, I changed it some PRs ago to match the CLI, but all other calls to `_tx` have `1.05`

Reply:
Gotcha